packages/css-engine/src/core/compare-media.ts

```diff
@@ -9,18 +9,19 @@ export const compareMedia = (
   optionB: MediaRuleOptions
 ) => {
   // Ensures a media with no min/max is always first
```

Comment:
<img width="296" alt="Screenshot 2023-05-01 at 16 59 05" src="https://user-images.githubusercontent.com/825702/235462561-be04d728-570b-4487-a6d8-827dda4ca2b4.png">

With these values it's not first

Reply:
actually we might want to us @taylornowotny if he things we should preserve the order in the top bar here as well

Reply:
yes please sort this list just like the breakpoints list in the top bar is sorted thank you

Reply:
that's the case